gtsam/hybrid/HybridGaussianConditional.h

```diff
@@ -150,9 +150,15 @@ class GTSAM_EXPORT HybridGaussianConditional
   /// Returns the continuous keys among the parents.
   KeyVector continuousParents() const;
 
-  /// The log normalization constant is max of the the individual
-  /// log-normalization constants.
-  double logNormalizationConstant() const override { return -logConstant_; }
+  /**
+   * @brief Return log normalization constant in negative log space.
+   *
+   * The log normalization constant is the min of the individual
+   * log-normalization constants.
+   *
+   * @return double
+   */
+  inline double negLogConstant() const override { return logConstant_; }
```

Comment:
Ok, almost done! but this should probably los be renamed to negLogConstant_ no?

Reply:
Yes I agree. Sorry, I need to be more thorough.